main.go

```diff
@@ -86,6 +88,11 @@ type Options struct {
 	GoogleOAuthClientCredentialFile    string
 	PinnedDateTime                     string
 
+	DaemonServer           bool
+	CommentProcessing      bool
+	ExcludeReposCommenting []string
+	IncludeReposCommenting []string
```

Comment:
Sorry, but I added another option so we don't comment by default in case we run locally or are testing deployment...  But yeah could be a good thing to consolidate one day.